src/main/java/com/google/firebase/messaging/AndroidNotification.java

```diff
@@ -309,6 +414,185 @@ public Builder setImage(String imageUrl) {
       return this;
     }
 
+    /**
+     * Sets the "ticker" text, which is sent to accessibility services. Prior to API level 21
+     * (Lollipop), sets the text that is displayed in the status bar when the notification 
+     * first arrives.
+     *
+     * @param ticker Ticker name.
+     * @return This builder.
+     */
+    public Builder setTicker(String ticker) {
+      this.ticker = ticker;
+      return this;
+    }
+
+    /**
+     * Sets the sticky flag. When set to false or unset, the notification is automatically 
+     * dismissed when the user clicks it in the panel. When set to true, the notification 
+     * persists even when the user clicks it.
+     *
+     * @param sticky The sticky flag
+     * @return This builder.
+     */
+    public Builder setSticky(boolean sticky) {
+      this.sticky = sticky;
+      return this;
+    }
+
+    /**
+     * Sets the time that the event in the notification occurred in milliseconds. Notifications
+     * in the panel are sorted by this time. The time will be formated in RFC3339 UTC "Zulu" 
```

Comment:
Suggest "is formatted"

(avoid "will")

Reply:
Done.